src/components/common/view/UserAvatar.js

```diff
@@ -58,38 +59,38 @@ const UserAvatar = (props: AvatarProps) => {
     </View>
   ) : (
     <View style={styles.avatar}>
-      <View style={styles.innerAvatar}>
-        <Avatar size={originalSize ? cropSize : 120} {...props} source={profile.avatar}>
+      <View style={[styles.innerAvatar, containerStyle]}>
+        <Avatar size={originalSize ? cropSize : normalize(136)} {...props} source={profile.avatar}>
```

Comment:
shouldn't need normalize here I think since the default value is not using normalzie

Reply:
Done